.github/workflows/release.yml

```diff
@@ -1,8 +1,8 @@
 name: Releases
 on:
-  push:
-    branches:
-      - main
+  workflow_dispatch:
+  schedule:
+    - cron:  '00 00 * * *'
 
```

Comment:
I don't believe this is the action we want to CRON.

The problem with this action is if all we have in the change is `chore` commits (which is accurate for profile updates), the changelog doesn't show a version bump so nothing is actually triggered. Refer to [this log](https://github.com/EddieHubCommunity/LinkFree/actions/runs/3042239940/jobs/4900198039) for an example.

The CRON should be on the dispatch workflow instead.

Reply:
Good point however the deploy only deploys releases, so that would not work either 🤔

Reply:
The deploy was the script we were running to trigger a manual deployment. We cannot CRON the release action because it will exit if it doesn't see a version-changing commit.